Documentation/ceph-object.md

```diff
@@ -224,19 +225,32 @@ To test the `CephObjectStore` we will install the `s3cmd` tool into the toolbox
 yum --assumeyes install s3cmd
 ```
 
+Configure s3cmd `s3cmd --configure` with the ACCESS_KEY and other details as described here - https://github.com/s3tools/s3cmd
+
+
+### List the generated bucket name
+
+Note that to keep the bucket name unique we created the bucket with `generateBucketName`.
+Please see here for more details regarding OBC creation [Object Bucket Claim](ceph-object-bucket-claim.html)
+
+```console
+s3cmd ls                                                                           
+2020-10-15 05:56  s3://ceph-bkt-<xxx>
+```
+
 ### PUT or GET an object
 
 Upload a file to the newly created bucket
 
 ```console
 echo "Hello Rook" > /tmp/rookObj
-s3cmd put /tmp/rookObj --no-ssl --host=${AWS_HOST} --host-bucket=  s3://rookbucket
```

Comment:
@alexcpn not sure this one is a required change, even with OBC it won't be ceph-bkt-<xxx>. Users can create bucket from cephobjectstore user as well. IMO better to keep `rookbucket` than `ceph-bkt-<xxx>`